src/modules/portfolio/components/portfolio-reports/portfolio-reports.jsx

```diff
@@ -64,7 +64,7 @@ export default class PortfolioReports extends Component {
       reportingFees,
     } = this.props
     let disableClaimReportingFeesNonforkedMarketsButton = ''
-    if (reportingFees.unclaimedEth.formatted === '-' && reportingFees.unclaimedRep.formatted === '-') {
+    if ((isNaN(reportingFees.unclaimedEth.formatted) || reportingFees.unclaimedEth.formatted === '') && (isNaN(reportingFees.unclaimedRep.formatted) || reportingFees.unclaimedRep.formatted === '')) {
       disableClaimReportingFeesNonforkedMarketsButton = 'disabled'
```

Comment:
not sure we want to use isNaN(...) with formatted number b/c isNaN('1,000') is true and formatted can have commas. probably want to user formattedValue

Reply:
zero is allowed here. because of that rounding thing